lib/takeaway.rb

```diff
@@ -0,0 +1,44 @@
+require_relative 'messager'
+require_relative 'menu'
+require_relative 'order'
+
+class Takeaway
+
+  attr_reader :basket
+
+
+  def initialize(menu, messager)
```

Comment:
Define classes in arguments menu = Menu.new https://github.com/makersacademy/takeaway-challenge/blob/master/docs/review.md  Dependency Injection